quarkus.ls.ext/com.redhat.quarkus.ls/src/test/java/com/redhat/quarkus/ls/JavaTextDocumentSnippetRegistryTest.java

```diff
@@ -23,6 +23,7 @@
 import org.eclipse.lsp4mp.ls.commons.snippets.ISnippetContext;
 import org.eclipse.lsp4mp.ls.commons.snippets.Snippet;
 import org.eclipse.lsp4mp.ls.commons.snippets.SnippetRegistry;
+import org.eclipse.lsp4mp.snippets.JavaSnippetCompletionContext;
```

Comment:
Is it this change is part of this PR?

Reply:
No, but the tests were failing locally if I didn't have this. However, it fails on GitHub Actions if I make this change, so I'll remove it